myhoard/controller.py

```diff
@@ -89,6 +95,21 @@ def key(backup):
     return sorted((backup for backup in backups if backup["completed_at"]), key=key)
 
 
+DELETE_NOT_REQUESTED_AT: str = datetime.datetime(2400, 1, 1, tzinfo=datetime.timezone.utc).isoformat()
+
+
+def sort_purgeable_backups(backups: List[Backup]) -> List[Backup]:
+    def key(backup: Backup) -> str:
+        assert backup["completed_at"] is not None
+        delete_requested_at: str = (
+            backup["delete_requested_at"] if backup["delete_requested_at"] is not None else DELETE_NOT_REQUESTED_AT
+        )
+        completed_at: str = datetime.datetime.fromtimestamp(backup["completed_at"]).isoformat()
+        return f"{delete_requested_at}__{completed_at}"
```

Comment:
The problem is that it sorts ascending. Because my sentinel date was before the current date, all of the values without a `deleete_requested_at` value ended up sorting at the front of the list, which is the opposite of what I wanted. It's not a random value, it's literally the year 2400. I originally had 1970. 

I want dates to sort. That's the whole point of the ISO datetimestamp, it's lexicographically sortable.